src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -71,8 +71,8 @@
     <Compile Include="$(CommonTestPath)\System\ObjectCloner.cs">
       <Link>Common\System\ObjectCloner.cs</Link>
     </Compile>
-    <Compile Include="$(CommonTestPath)\System\Collections\DictionaryExtensions.cs">
-      <Link>Common\System\Collections\DictionaryExtensions.cs</Link>
```

Comment:
There is around 7 other projects that use `DictionaryExtensions.cs` file.

That is why CI is failing. You could verify this by running:
`build.cmd`
`build-tests.cmd -skiptests`

Can we leave this file as it was and add the new API there as well? Since there is a lot of test projects that use this file.